src/custom/lib/hooks/useCurrencyLogoURIs/useCurrencyLogoURIsMod.ts

```diff
@@ -66,7 +66,7 @@ export default function useCurrencyLogoURIs(currency?: Currency | null): string[
         // mod
         const logoURI = getOverriddenTokenLogoURI(currency)
         if (logoURI) {
-          logoURIs.push(logoURI)
+          return [logoURI]
```

Comment:
1. We got `WXDAI` token from `1HIVE` tokens list with an outdated `logoURL`
2. We have an override for `WXDAI` in `ADDRESS_IMAGE_OVERRIDE`
3. Due to the race conditions we got the outdated `logoURL`

Reply:
Nice find!